dpcpp/base/executor.dp.cpp

```diff
@@ -274,15 +274,38 @@ void DpcppExecutor::set_device_property(dpcpp_queue_property property)
     }
     this->get_exec_info().max_workgroup_size = static_cast<int>(
         device.get_info<sycl::info::device::max_work_group_size>());
+// They change the max_work_item_size with template parameter Dimension after
+// major version 6 and adding the default = 3 is not in the same release.
```

Comment:
would 3 be a sensible default at all? It sounds like that is a pretty CUDA-specific view? maybe just state that this is the dimensionality we use.